### PR TITLE
PLANET-7080 Convert High Level Topic layout into a Block Template

### PR DESCRIPTION
**Pull request by mleray**
### Description

See [https://jira.greenpeace.org/browse/PLANET-7080](https://jira.greenpeace.org/browse/PLANET-7080)
This PR also removes the `CarouselHeader`, `GravityFormWithText` and `Covers` templates that are no longer needed

### Testing

Either on local or on the test instance, you should still be able to add the `High-Level Topic` pattern layout, and it should look the same as before.

### CODE REVIEW

**Comment by planet-4** [comment]
### Test instance is ready :rocket:

:new_moon: [phoebe](https://www-dev.greenpeace.org/test-phoebe) | [admin](https://www-dev.greenpeace.org/test-phoebe/wp-admin/) | [blocks report](https://www-dev.greenpeace.org/test-phoebe/wp-admin/?page=plugin_blocks_report) | [CircleCI](https://app.circleci.com/pipelines/github/greenpeace/planet4-test-phoebe) | [composer-local.json](https://github.com/greenpeace/planet4-test-phoebe/blob/main/composer-local.json)

:watch: 2023.07.14 13:18:34

File: assets/src/block-templates/high-level-topic/template.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      mediaPosition: 'right',
+      alignFull: true,
+    }],
+    ['planet4-block-templates/highlighted-cta', {
+      titlePlaceholder: __('Featured action title', 'planet4-blocks'),
+    }],
+    ['planet4-blocks/covers', {
+      cover_type: 'take-action',
+      title: __('How you can help', 'planet4-blocks'),
+    }],
```
Comment (lithrel) [inline assets/src/block-templates/high-level-topic/template.js:32]: Missing an Articles block between the 2 covers I think ?

Comment (mleray) [reply assets/src/block-templates/high-level-topic/template.js:32]: Ah, sorry, my bad 🙈 fixed!

**Review by lithrel — APPROVED** [review]
:+1:

**Pull request merged**